### ISSUE TITLE
Installation fails when trying to install pandas

### ISSUE THREAD

**Issue by catalys1** [PyTorchLightning/lightning-flash#124]
## 🐛 Bug

When trying to install lightning-flash via `pip install lightning-flash`, I get an error caused by the dependency on `pandas==1.1.2`:
```
ERROR: Could not build wheels for pandas which use PEP 517 and cannot be installed directly
```
I get the same error when I try to install `pandas==1.1.2` independent of lightning-flash. I am able to install `pandas==1.2.2` without any problems. I'm using Python 3.9.1 and Ubuntu 20.04.

Is it necessary to have the pandas version fixed at 1.1.2?

### Expected behavior

Installation without any problems

### Environment

 - PyTorch Version (e.g., 1.0): `1.7`
 - OS (e.g., Linux): `Ubuntu 20.04`
 - How you installed PyTorch (`conda`, `pip`, source): `pip`
 - Build command you used (if compiling from source):
 - Python version: `3.9.1`
 - CUDA/cuDNN version: `10.2`
 - GPU models and configuration: `1080TI`
 - Any other relevant information:

**Comment by Borda** [comment]
I think that this would be also solved by #62
cc: @carmocca

**Comment by kaushikb11** [comment]
It's a python 3.9 issue.
@Borda https://github.com/PyTorchLightning/lightning-flash/pull/125 It would fix it for now.
